README.md

```diff
@@ -166,6 +166,96 @@ $method = new Payment\Get('cb5e1c95-7685-4499-a2b1-ae0f28297b92');
 $payment = $client->call($method);
 ```
 
+================
+### Payment Link
```

Comment:
All the payment link docs should be moved to `docs/README.md`, as this file contains docs for all the other methods, except for Payment